### PR TITLE
Add email registration form for the 2022 conference

### PR DESCRIPTION
**Pull request by olivierobert**
## What happened

Reworked the email registration page.
 
## Insight

All new registration will be tagged with `conference-2022` and be accessible via the segment `Conference 2022 Form Subscribers`:

Screen Shot 2565-07-17 at 14 21 10

image
 
## Proof Of Work

|Small Screen|Large Screen|
|-|-|
|image |image|

### CODE REVIEW

**Comment by citizen428** [comment]
This looks good, thanks @olivierobert!

File: index.md [hunk 1]
```diff
@@ -4,14 +4,15 @@ title: Home
 permalink: /
 ---
 
-<section class="cancellation-announcement">
-  <p>Due to the uncertainty surrounding COVID-19 we have decided to reschedule the next RubyConfTH, originally planned for October 2020, to a later date in 2021 to allow us to prepare the best possible conference for you.</p>
-  <p>Bangkok.rb plans to organize a variety of online and, once possible, offline events in Bangkok for the Ruby community <br /> during the rest of 2020.</p>
+<section class="conference-announcement">
+  <p>A comeback of RubyConf TH for 2022 is in the works 🎉</p>
+  <p>The conference is scheduled to take place on December 10-11th 2022 at <a href="https://www.pullmanbangkokkingpower.com/">Pullman Bangkok King Power</a> in Bangkok</p>
```
Comment (matthewmayer) [inline index.md:9]: Bangkok, Thailand

(Thailand is not mentioned on page otherwise so good for SEO)

File: index.md [hunk 2]
```diff
@@ -4,14 +4,15 @@ title: Home
...
+  <p>Standard ticket price is <mark>3,200 THB</mark> for the two-day event inclusive of an international buffet on both days and a welcome party 🥳</p>
```
Comment (matthewmayer) [inline index.md:10]: (Approx 89 USD)

inclusive of all talks, an international lunch buffet on both days and a ticket to the official party

**Review by matthewmayer — CHANGES_REQUESTED** [review]
Also update metadata to 2022

File: index.md (hunk 2 above)
Comment (olivierobert) [reply index.md]: All changed in [d0069b6](https://github.com/bangkokrb/rubyconfth/pull/174/commits/d0069b693e9dad51b2ce1a117901f360d12fc1c7)

**Comment by olivierobert** [comment]
Netlify Deploy previews only works when cleaning the cache and redeploying 🤦

**Pull request merged**